pandas/_libs/tslibs/strptime.pyx

```diff
@@ -281,6 +290,32 @@ def array_strptime(ndarray[object] values, object fmt,
                         else:
                             tz = value
                             break
+            elif parse_code == 19:
```

Comment:
can you move this whole parse to a function and just all it here (and return the values as a tuple)